src/service.js

```diff
@@ -428,7 +428,7 @@ module.exports = function(mixinOptions) {
 						}),
 					);
 
-					this.graphqlHandler = this.apolloServer.createHandler();
+					this.graphqlHandler = this.apolloServer.createHandler(mixinOptions.serverOptions);
```

Comment:
Can you explain this change?  I couldn't track down where the `apolloServer.createHandler()` was specified and I'm trying to understand why it would need to be specified in both the constructor and the call to `createHandler`.